packages/superset-ui-chart/test/components/reactify.test.jsx

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { mount } from 'enzyme';
+import reactify from '../../src/components/reactify';
```

Comment:
super nit -- if you import from `../../src` we'd potentially also get linting to ensure that it's always exported.

Reply:
cool